project-koku/koku
# Adding unleash token to docker file

This change will fix local unleash

## Testing

1. Checkout Branch
2. add UNLEASH_TOKEN to .env file 
3. Restart Koku
4. See no Unleash errors in logs

## Notes

...
